pkg/specs/pvcs.go

```diff
@@ -131,54 +145,92 @@ func DetectPVCs(
 			continue
 		}
 
+		// TODO optimize
+		// we save all the instance we found so far
+		instanceName := pvc.Labels[utils.InstanceLabelName]
```

Comment:
We need to make it backward compatible and, if we don't have that label name, look at a Pod with the name of the PVC. Or, maybe, we could retag the PVC before...